rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -19,6 +21,40 @@
  */
 public final class RxView {
   /**
+   * Create an observable of timestamps for {@code view}'s attaches.
```

Comment:
"Create an observable which emits on {@code view} attach events. The emitted value is unspecified and should only be used as notification."